### PR TITLE
Feature/add release attributes

### PR DESCRIPTION
**Pull request by PatrickConnollyONS**
### What

Add WelshStatistic and Survey attributes to the Release type for zebedee and releasecalendar
Some small tidy ups

### How to review

Review code and ensure test are passing

### Who can review

Anyone

### CODE REVIEW

File: zebedee/client.go [hunk 1]
```diff
@@ -192,7 +182,7 @@ func (c *Client) put(ctx context.Context, userAccessToken, path string, payload
 }
 
 // GetBreadcrumb returns a Breadcrumb
-func (c *Client) GetBreadcrumb(ctx context.Context, userAccessToken, collectionID, lang, uri string) ([]Breadcrumb, error) {
```
Comment (rafahop) [inline zebedee/client.go:195]: If we do not need these params, should we remove them?

File: zebedee/data.go [hunk 2]
```diff
@@ -134,10 +136,6 @@ type Alert struct {
 	Type     string `json:"type"`
 }
 
-// Related stores the Title and URI for any related data (e.g. related publications on a dataset page)
-// Deprecated: use Link
-type Related = Link
```
Comment (rafahop) [inline zebedee/data.go:139]: Is this a breaking change?

Comment (PatrickConnollyONS) [reply zebedee/data.go:139]: yes it is - I have removed it

File: zebedee/client.go (hunk 1 above)
Comment (PatrickConnollyONS) [reply zebedee/client.go:195]: yes I think so. However I have reverted most of these 'tidy-ups' as they are introducing breaking changes

**Pull request merged**